eng/helix/Helix.proj

```diff
@@ -117,15 +117,31 @@
   <ItemGroup>
     <AdditionalDotNetPackageFeed Include="https://dotnetbuilds.blob.core.windows.net/internal"
                                  Condition="'$(SYSTEM_TEAMPROJECT)' == 'internal'">
-      <SasToken>$(DotNetBuildsInternalContainerReadToken)</SasToken>
+      <SasToken>$([System.Environment]::GetEnvironmentVariable('DotNetBuildsInternalContainerReadToken'))</SasToken>
```

Comment:
Not an accident. It needs to be passed by environment so that the value is not escaped nor split (passing by command line was causing the value to be split along `&` characters when trying to use the non-encoded variant).